core/templates/pages/exploration-editor-page/editor-tab/templates/modal-templates/add-hint-modal.component.html

```diff
@@ -4,8 +4,7 @@ <h3>Add Hint</h3>
 </div>
 
 <div class="modal-body">
-  <form name="addHintForm.hintTextForm"
-        class="form-inline">
+  <form class="form-inline">
```

Comment:
Why is this modified?

Reply:
The name property is not needed here we are not validating the form, or using any other form properties.